<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TasksDialog</class>
 <widget class="QDialog" name="TasksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Confirm tasks</string>
  </property>
  <property name="windowIcon">
   <iconset theme="utilities-terminal">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TasksView" name="tasksView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="noConfirmCheckBox"/>
   </item>
   <item>
    <widget class="QCheckBox" name="forceCheckBox"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="afterCompletionlabel"/>
     </item>
     <item>
      <widget class="QComboBox" name="afterCompletionComboBox"/>
     </item>
     <item>
      <spacer name="afterCompletionSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="commandsEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Commands</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TasksView</class>
   <extends>QTreeView</extends>
   <header>src/tasks-view/tasksview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TasksDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TasksDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noConfirmCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>TasksDialog</receiver>
   <slot>setNoConfirm(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forceCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>TasksDialog</receiver>
   <slot>setForce(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>afterCompletionComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>TasksDialog</receiver>
   <slot>setAfterCompletion(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setNoConfirm(bool)</slot>
  <slot>setForce(bool)</slot>
  <slot>setAfterCompletion(int)</slot>
 </slots>
</ui>
